Sources/Fullscreen/Consent/Details/ConsentDetailView.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  Copyright © 2018 FINN AS. All rights reserved.
+//
+import UIKit
+
+public protocol ConsentDetailViewDelegate: class {
+    func consentDetailView(_ consentDetailView: ConsentDetailView, didToggleSwitch position: Bool)
+    func readMoreButtonPressed(in consentDetailView: ConsentDetailView)
```

Comment:
I think the correct naming for methods like this one without attributes would be:

```swift
func consentDetailViewReadMoreButtonPressed(_ consentDetailView: ConsentDetailView)
```

Reply:
Not according to tableViewDelegate methods...

Reply:
Why apple, why??